_includes/life_cycle.html

```diff
@@ -28,7 +28,48 @@
 {% elsif site.life_cycle == "stable" %}
 
 {% comment %}
-We do not do anything special unless this is a Carpentries Lab lesson
+We do not do anything special for lessons in stable
+{% endcomment %}
+
+
+{% comment %}
+Below we cover the 3 phases of lesson transition towards the Carpentries Workbench:
+- transition_step_1 (`transition_url` variable needed): we indicate that there
+is a new version of the lesson that is available for testing, and that it will be deprecated in the future.
+- transition_step_2 (`transition_url` and `transition_date` variables needed): we indicate that a new version of the lesson is avaiable with a deprecation date.
+- transition_step_3 (`transition_url` variable needed): the lesson if fully deprecated, and a redirect is set up to the new URL
+{% endcomment %}
+
+{% elsif site.life_cycle == "transition_step_1" %}
+{% include check_transition_variables.html need_transition_date = 'false' %}
+
+<div class="alert alert-warning life-cycle">
+  A newer version of this lesson is being tested:
+  <a href="{{ site.transition_url }}">{{ site.transition_url }}</a>.
+  This lesson will become deprecated in the near future.
```

Comment:
I think saying "this lesson will become deprecated" evokes the feeling that the lesson itself will be going away as opposed to being presented in a different form. 

```suggestion
  The Jekyll version of this lesson will become deprecated in the near future.
```